12-01-rerere/12-01-rerere.en.sbv

```diff
@@ -8,22 +8,22 @@ have to be resolved once again. For example
 we start to rebase. Place some commits over commits.
 
 0:00:09.045,0:00:12.015
-Resolve conflicts from them. And here happens that we have to --abort the rebase.
+Resolve conflicts from them. And here happens that we have to '--abort' the rebase.
 
 0:00:12.015,0:00:14.705
- Thus when we rebase again
+Thus when we rebase again
 
 0:00:14.705,0:00:17.445
 will emerge the same conflicts. 

```

Comment:
"Later": word order, like: same conflicts will emerge.
Here and further I just state the things we'll have to fix later. For now its ok